library/src/com/google/maps/android/geojson/GeoJsonLayer.java

```diff
@@ -0,0 +1,251 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLngBounds;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+
+/**
+ * A class that allows the developer import GeoJSON data, style it and interact with the imported
```

Comment:
I think this class is complex enough that you should include a {@code} block showing a code sample of how to use it in the JavaDoc here.